<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParsingTool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QPushButton" name="interface_but">
      <property name="text">
       <string>Настройки интерфейса</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="search_forms">
      <property name="text">
       <string>Поиск по формам слова</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QTextEdit" name="result_UI"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="word_line"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="hist_but">
      <property name="text">
       <string>История поиска</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="url_line"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите слово для поиска:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="show_pics">
      <property name="text">
       <string>Просмотреть картинки </string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="search_exact">
      <property name="text">
       <string>Точный поиск</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="save_but">
      <property name="text">
       <string>Сохранить результат в txt файл</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите адрес сайта (адреса нескольких сайтов через пробел):</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="check_pics">
      <property name="text">
       <string>Загружать изображения?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
